### ISSUE TITLE
Machine Learning training

### ISSUE THREAD

**Issue by francescacirillo85-wq** [francescacirillo85-wq/Vari-progetti#1]
Progetto di machine learning training che usa la regressione logistica per addestrare il modello su un training set di 800 unità e testare il modello su un validation set di 200 unità.

[Machine_learning_training.ipynb](https://github.com/user-attachments/files/23968178/Machine_learning_training.ipynb)
